src/task.ts

```diff
@@ -33,8 +33,11 @@ class Task implements TaskType {
     this.executions = task.getExecutionsList().map(execution => {
        return {
          epoch: execution.getEpoch(),
-         userOpHash: execution.getUserOpHash(),
```

Comment:
we no longer has userOpHash. 

We just call it outputData as a generic name for outcome (result) of a node execution.

Reply:
👌